# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: 249eede42e9f
# Full Hash: 249eede42e9fdd7b58f98b6498fc1a104505fc5f
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2020-07-11 21:52:53
# Regressor Bug: 1557983
# File Overlap Count: 1
# Description:
#   Bug 1557983. Remove nsPrintJob::HasPrintCallbackCanvas.
#   
#   After refactoring the 'beforeprint'/'afterprint' code, this method is not used
#   any more.
#   
# ==============================================================================

diff -r dfbaf61ce16a -r 249eede42e9f layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Sat Jul 11 09:06:25 2020 +0000
+++ b/layout/printing/nsPrintJob.h	Sat Jul 11 09:10:03 2020 +0000
@@ -131,12 +131,6 @@
 
   void TurnScriptingOn(bool aDoTurnOn);
 
-  /**
-   * Checks to see if the document this print engine is associated with has any
-   * canvases that have a mozPrintCallback.
-   * https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement#Properties
-   */
-  bool HasPrintCallbackCanvas() { return mHasMozPrintCallback; }
   bool PrePrintPage();
   bool PrintPage(nsPrintObject* aPOect, bool& aInRange);
   bool DonePrintingPages(nsPrintObject* aPO, nsresult aResult);
